### ISSUE TITLE
Disappearing botania sparks

### ISSUE THREAD

**Issue by Copyoflegos** [NillerMedDild/Enigmatica4#319]
**Modpack version**
0.4.8

**Issue**
When logging out of multiplayer server botania sparks will disappear completely when I log back in.

**Logs**
N/A

**Comment by NielsPilgaard** [comment]
Thank you for the report, I will investigate the issue.

**Comment by neochiken** [comment]
I can confirm this, it seems to occur whenever a player changes dimensions and then reloads the area,
I am also getting permanent 0 light level issues with botania on dimension reload
Note: my botania setup in inside of a colony, don't know if this could possibly be causing the issue

**Comment by NielsPilgaard** [comment]
@Copyoflegos was your spark also inside a colony?

**Comment by Copyoflegos** [comment]
Nope, the sparks weren't in any specific area as far as I can tell. They were just placed over mana pools and the terra steel former plate.

**Comment by NielsPilgaard** [comment]
I have reported the issue here: https://github.com/Vazkii/Botania/issues/3224
Would you mind following it, so you can answer questions that could pop up? :)
